lib/src/pages/leader_board_page.dart

```diff
@@ -15,8 +17,53 @@ class LeaderBoardPage extends StatefulWidget {
 }
 
 class _LeaderBoardPageState extends State<LeaderBoardPage> {
+  bool _initilized = false;
+
+  Map<String, UserModel> users = {};
+  List<MapEntry<String, int>> userScoresList = [];
+
+  void getData() async {
+    users = await UserService.instance.asMap();
+    Map<String, int> scoresMappedToUId =
+        await ScoreService.instance.mappedToUid();
+
+    userScoresList = scoresMappedToUId.entries.toList()
+      ..sort(
+        (a, b) => b.value.compareTo(a.value),
+      );
+
+    setState(() {});
```

Comment:
der Schönheit halber sollte initialized=true im setState stehen. Insofern du _initialized auch in der UI verwendest läufst du ansonsten auch noch Gefahr eine Race Condition aufzustellen bei der es mal schon gesetzt ist, mal nicht.